README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+use of HTML to reinforce the meaning of content on a web page, rather than merely define its appearance. Semantic Elements describe the meaning and the content to the browser.
```

Comment:
Great answer! You nailed it on the head! Semantic HTML provides information about the contents of the page and is essential for search engine optimizations